library/encoding.go

```diff
@@ -137,7 +137,12 @@ func DecodeSymmetric(messageJSON string, symmetricKey string) (string, error) {
 	}
 
 	if msg.Version == 0 {
-		return marshalJSON(msg.Payload)
+		return marshalJSON(struct {
```

Comment:
Better ti define this struct as type and use it as i see 2 copies of same definition.